tests/script/test_realm.py

```diff
@@ -149,3 +201,50 @@ async def test_realm_realmDestroyed_sandbox(websocket, context_id):
             "realm": ANY_STR,
         }
     } == response
+
+
+@pytest.mark.asyncio
+async def test_realm_realmDestroyed_worker(websocket, context_id, html):
+    worker_url = 'data:application/javascript,while(true){}'
+    url = html(f"<script>window.w = new Worker('{worker_url}');</script>")
+
+    await subscribe(websocket, ["script.realmDestroyed"])
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "context": context_id,
+                "url": url,
+                "wait": "complete",
+            }
+        })
+
+    assert {
```

Comment:
why is the realm destroyed event expected here?

Reply:
is it the previous realm in the page? can we in some way assert that the right realm is destroyed?

Reply:
It's the previous realm. It's guaranteed that this will always be the realm of the previous page.